src/main/java/org/zalando/nakadi/config/ValidatorConfig.java

```diff
@@ -65,6 +66,7 @@ public SchemaEvolutionService schemaEvolutionService() throws IOException {
         final Schema metaSchema = SchemaLoader.load(metaSchemaJson);
 
         final List<SchemaEvolutionConstraint> schemaEvolutionConstraints = Lists.newArrayList(
+                new CategoryChangeConstraint(),
```

Comment:
@andrey I agree with you, but refactoring it now would mean to change some 15 classes just to add this one more.